### ISSUE TITLE
Add locale support

### ISSUE THREAD

**Issue by samo79** [sodero/MUI-Vim#6]
Implementing locale support in Amiga style

**Comment by sodero** [comment]
Indeed, this would be a good thing to have. I need to figure out how Vim does these things.

**Comment by polluks** [comment]
FlexCat may help https://yam.ch/discussion/topic/116?discussion_action=set-display;display=flat-asc

**Comment by polluks** [comment]
My lang.rexx
````
/* Set Unix language environment */
address command
"rxset LANG `search from MOSSYS:Data/ISO/639 $Language`"
parse value getclip(LANG) with . country .
ask '"Save LANG='country'?"'
if rc=0 then SAVE=""
"setenv LANG" SAVE country
````

**Comment by sodero** [comment]
Nice, it looks like Vim uses gettext for localization. I don't know gettext and I need to do some investigations to see whether it's feasible to create a shim or if the (Vim) native solution is preferable. One obvious pro of using Vims solution would be that we get many many translations for free, and that those are updated by upstream.
